Docker/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM node:16-alpine
+FROM node:17-alpine3.14
```

Comment:
i think we want to stay on 16 as it is the LTS version. we've had other bugs pop up by running 17 before.

Reply:
That's quite interesting. Pairing to your message below, I am building it in the "wrong" way. Now, I build the container locally and then use on the composer. Really appreciate the feedback!